bib/cmd/bootc-image-builder/main.go

```diff
@@ -348,7 +340,7 @@ func run() error {
 	rootCmd.AddCommand(manifestCmd)
 	manifestCmd.Flags().String("rpmmd", "/rpmmd", "rpm metadata cache directory")
 	manifestCmd.Flags().String("config", "", "build config file")
-	manifestCmd.Flags().String("type", "qcow2", "image type to build [qcow2, ami]")
+	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, "image type to build [qcow2, ami]")
```

Comment:
s/type/types/ right?

Reply:
You mean on the description `image types to build,,,`
We still wan to use `--type qcow2 --type raw --type iso`
I think `--type qcow2,raw,iso` also works.